[2023-12-03T23:08:08.920+0000] {processor.py:157} INFO - Started process (PID=3770) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:08:08.921+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:08:08.922+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:08:08.921+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:08:08.926+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:08:08.925+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:08:08.926+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:08:08.926+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:08:08.926+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:08:08.938+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:08:58.823+0000] {processor.py:157} INFO - Started process (PID=3819) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:08:58.824+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:08:58.825+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:08:58.825+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:08:58.829+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:08:58.828+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:08:58.829+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:08:58.829+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:08:58.830+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:08:58.841+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:09:45.397+0000] {processor.py:157} INFO - Started process (PID=3870) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:09:45.399+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:09:45.400+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:09:45.399+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:09:45.404+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:09:45.403+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:09:45.404+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:09:45.404+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:09:45.404+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:09:45.416+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:10:33.583+0000] {processor.py:157} INFO - Started process (PID=3924) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:10:33.584+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:10:33.585+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:10:33.585+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:10:33.589+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:10:33.588+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:10:33.589+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:10:33.589+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:10:33.589+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:10:33.601+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:11:19.471+0000] {processor.py:157} INFO - Started process (PID=3974) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:11:19.473+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:11:19.474+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:11:19.473+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:11:19.477+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:11:19.477+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:11:19.478+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:11:19.478+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:11:19.478+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:11:19.489+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:12:05.142+0000] {processor.py:157} INFO - Started process (PID=4024) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:12:05.143+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:12:05.144+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:12:05.144+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:12:05.148+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:12:05.147+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:12:05.148+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:12:05.148+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:12:05.149+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:12:05.161+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03T23:12:49.732+0000] {processor.py:157} INFO - Started process (PID=4073) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:12:49.733+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:12:49.734+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:12:49.734+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:12:49.738+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:12:49.737+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:12:49.738+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:12:49.738+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:12:49.738+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:12:49.749+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:13:37.407+0000] {processor.py:157} INFO - Started process (PID=4126) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:13:37.408+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:13:37.409+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:13:37.409+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:13:37.413+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:13:37.413+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:13:37.414+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:13:37.413+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:13:37.414+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:13:37.425+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:14:24.525+0000] {processor.py:157} INFO - Started process (PID=4176) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:14:24.526+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:14:24.527+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:14:24.527+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:14:24.531+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:14:24.530+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:14:24.531+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:14:24.531+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:14:24.532+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:14:24.543+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:15:13.769+0000] {processor.py:157} INFO - Started process (PID=4231) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:15:13.770+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:15:13.771+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:15:13.770+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:15:13.775+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:15:13.774+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:15:13.775+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:15:13.775+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:15:13.775+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:15:13.786+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:16:02.432+0000] {processor.py:157} INFO - Started process (PID=4283) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:16:02.433+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:16:02.434+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:16:02.433+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:16:02.438+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:16:02.437+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:16:02.438+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:16:02.438+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:16:02.438+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:16:02.450+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:16:49.273+0000] {processor.py:157} INFO - Started process (PID=4337) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:16:49.274+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:16:49.275+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:16:49.275+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:16:49.279+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:16:49.278+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:16:49.279+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:16:49.279+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:16:49.279+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:16:49.291+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:17:35.437+0000] {processor.py:157} INFO - Started process (PID=4390) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:17:35.438+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:17:35.439+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:17:35.438+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:17:35.443+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:17:35.442+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:17:35.443+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:17:35.443+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:17:35.443+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:17:35.455+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:18:21.488+0000] {processor.py:157} INFO - Started process (PID=4440) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:18:21.489+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:18:21.490+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:18:21.490+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:18:21.494+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:18:21.493+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:18:21.494+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:18:21.494+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:18:21.494+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:18:21.506+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:19:04.414+0000] {processor.py:157} INFO - Started process (PID=4490) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:19:04.415+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:19:04.416+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:19:04.416+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:19:04.420+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:19:04.419+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:19:04.420+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:19:04.420+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:19:04.421+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:19:04.432+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:19:49.708+0000] {processor.py:157} INFO - Started process (PID=4539) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:19:49.709+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:19:49.710+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:19:49.710+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:19:49.714+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:19:49.713+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:19:49.714+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:19:49.714+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:19:49.714+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:19:49.726+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:20:31.974+0000] {processor.py:157} INFO - Started process (PID=4587) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:20:31.976+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:20:31.977+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:20:31.976+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:20:31.980+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:20:31.980+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:20:31.981+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:20:31.981+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:20:31.981+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:20:31.993+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:21:17.697+0000] {processor.py:157} INFO - Started process (PID=4639) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:21:17.698+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:21:17.699+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:21:17.699+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:21:17.703+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:21:17.702+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:21:17.703+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:21:17.703+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:21:17.703+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:21:17.715+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:22:08.587+0000] {processor.py:157} INFO - Started process (PID=4693) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:22:08.589+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:22:08.590+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:22:08.589+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:22:08.593+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:22:08.593+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:22:08.594+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:22:08.593+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:22:08.594+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:22:08.605+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:22:55.458+0000] {processor.py:157} INFO - Started process (PID=4747) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:22:55.460+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:22:55.461+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:22:55.461+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:22:55.465+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:22:55.464+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:22:55.465+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:22:55.465+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:22:55.465+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:22:55.476+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:23:41.269+0000] {processor.py:157} INFO - Started process (PID=4797) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:23:41.270+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:23:41.271+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:23:41.271+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:23:41.275+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:23:41.274+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:23:41.275+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:23:41.275+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:23:41.275+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:23:41.287+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:24:28.411+0000] {processor.py:157} INFO - Started process (PID=4848) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:24:28.413+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:24:28.414+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:24:28.413+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:24:28.417+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:24:28.417+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:24:28.418+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:24:28.418+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:24:28.418+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:24:28.430+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:25:18.476+0000] {processor.py:157} INFO - Started process (PID=4903) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:25:18.478+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:25:18.479+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:25:18.478+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:25:18.483+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:25:18.482+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:25:18.483+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:25:18.483+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:25:18.483+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:25:18.495+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:26:05.201+0000] {processor.py:157} INFO - Started process (PID=4954) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:26:05.202+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:26:05.203+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:26:05.203+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:26:05.207+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:26:05.206+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:26:05.207+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:26:05.207+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:26:05.207+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:26:05.219+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:26:51.754+0000] {processor.py:157} INFO - Started process (PID=5005) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:26:51.756+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:26:51.757+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:26:51.756+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:26:51.761+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:26:51.760+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:26:51.761+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:26:51.761+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:26:51.761+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:26:51.773+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03T23:27:33.956+0000] {processor.py:157} INFO - Started process (PID=5052) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:27:33.957+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:27:33.958+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:27:33.958+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:27:33.962+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:27:33.962+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:27:33.963+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:27:33.962+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:27:33.963+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:27:33.975+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:28:18.249+0000] {processor.py:157} INFO - Started process (PID=5101) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:28:18.251+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:28:18.252+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:28:18.251+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:28:18.256+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:28:18.255+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:28:18.256+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:28:18.256+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:28:18.256+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:28:18.267+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:29:05.964+0000] {processor.py:157} INFO - Started process (PID=5152) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:29:05.966+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:29:05.967+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:29:05.966+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:29:05.970+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:29:05.970+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:29:05.971+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:29:05.970+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:29:05.971+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:29:05.982+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:29:55.099+0000] {processor.py:157} INFO - Started process (PID=5208) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:29:55.101+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:29:55.101+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:29:55.101+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:29:55.105+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:29:55.105+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:29:55.106+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:29:55.105+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:29:55.106+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:29:55.117+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:30:41.436+0000] {processor.py:157} INFO - Started process (PID=5259) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:30:41.437+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:30:41.438+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:30:41.438+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:30:41.442+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:30:41.441+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:30:41.442+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:30:41.442+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:30:41.442+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:30:41.454+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:31:27.996+0000] {processor.py:157} INFO - Started process (PID=5309) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:31:27.997+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:31:27.998+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:31:27.998+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:31:28.002+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:31:28.001+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:31:28.002+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:31:28.002+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:31:28.002+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:31:28.014+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:32:14.879+0000] {processor.py:157} INFO - Started process (PID=5362) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:32:14.881+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:32:14.882+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:32:14.881+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:32:14.886+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:32:14.885+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:32:14.886+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:32:14.886+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:32:14.886+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:32:14.898+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:32:59.729+0000] {processor.py:157} INFO - Started process (PID=5411) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:32:59.730+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:32:59.731+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:32:59.730+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:32:59.735+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:32:59.734+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:32:59.735+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:32:59.735+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:32:59.735+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:32:59.746+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:33:46.093+0000] {processor.py:157} INFO - Started process (PID=5461) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:33:46.094+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:33:46.095+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:33:46.095+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:33:46.099+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:33:46.098+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:33:46.099+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:33:46.099+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:33:46.099+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:33:46.110+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:34:32.930+0000] {processor.py:157} INFO - Started process (PID=5512) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:34:32.932+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:34:32.933+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:34:32.932+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:34:32.936+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:34:32.936+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:34:32.937+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:34:32.937+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:34:32.937+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:34:32.948+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:35:22.884+0000] {processor.py:157} INFO - Started process (PID=5567) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:35:22.886+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:35:22.887+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:35:22.886+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:35:22.891+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:35:22.890+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:35:22.891+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:35:22.891+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:35:22.891+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:35:22.902+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:36:12.584+0000] {processor.py:157} INFO - Started process (PID=5620) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:36:12.585+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:36:12.586+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:36:12.586+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:36:12.590+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:36:12.589+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:36:12.590+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:36:12.590+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:36:12.590+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:36:12.601+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:37:02.905+0000] {processor.py:157} INFO - Started process (PID=5676) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:37:02.906+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:37:02.907+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:37:02.907+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:37:02.911+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:37:02.910+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:37:02.911+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:37:02.911+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:37:02.911+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:37:02.923+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:37:49.220+0000] {processor.py:157} INFO - Started process (PID=5727) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:37:49.221+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:37:49.222+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:37:49.222+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:37:49.226+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:37:49.225+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:37:49.226+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:37:49.226+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:37:49.226+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:37:49.238+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:38:37.597+0000] {processor.py:157} INFO - Started process (PID=5778) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:38:37.598+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:38:37.599+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:38:37.599+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:38:37.603+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:38:37.602+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:38:37.603+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:38:37.603+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:38:37.604+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:38:37.624+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-12-03T23:39:23.959+0000] {processor.py:157} INFO - Started process (PID=5828) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:39:23.960+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:39:23.961+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:39:23.961+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:39:23.965+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:39:23.964+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:39:23.965+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:39:23.965+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:39:23.966+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:39:23.977+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:40:09.282+0000] {processor.py:157} INFO - Started process (PID=5878) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:40:09.283+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:40:09.284+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:40:09.284+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:40:09.288+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:40:09.287+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:40:09.288+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:40:09.288+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:40:09.289+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:40:09.300+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:40:59.408+0000] {processor.py:157} INFO - Started process (PID=5934) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:40:59.409+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:40:59.410+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:40:59.410+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:40:59.414+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:40:59.414+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:40:59.415+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:40:59.414+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:40:59.415+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:40:59.427+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03T23:41:45.768+0000] {processor.py:157} INFO - Started process (PID=5985) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:41:45.769+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:41:45.770+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:41:45.770+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:41:45.774+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:41:45.774+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:41:45.775+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:41:45.774+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:41:45.775+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:41:45.786+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:42:33.278+0000] {processor.py:157} INFO - Started process (PID=6035) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:42:33.279+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:42:33.280+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:42:33.280+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:42:33.284+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:42:33.283+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:42:33.284+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:42:33.284+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:42:33.284+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:42:33.296+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:43:18.216+0000] {processor.py:157} INFO - Started process (PID=6085) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:43:18.217+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:43:18.218+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:43:18.218+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:43:18.222+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:43:18.221+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:43:18.222+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:43:18.222+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:43:18.223+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:43:18.234+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:44:08.255+0000] {processor.py:157} INFO - Started process (PID=6394) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:44:08.257+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:44:08.258+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:44:08.257+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:44:08.262+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:44:08.261+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:44:08.262+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:44:08.262+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:44:08.262+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:44:08.274+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03T23:44:50.980+0000] {processor.py:157} INFO - Started process (PID=6599) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:44:50.981+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:44:50.982+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:44:50.982+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:44:50.986+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:44:50.985+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:44:50.986+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:44:50.986+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:44:50.986+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:44:50.998+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:45:35.933+0000] {processor.py:157} INFO - Started process (PID=6648) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:45:35.934+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:45:35.935+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:45:35.935+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:45:35.939+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:45:35.938+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:45:35.939+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:45:35.939+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:45:35.940+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:45:35.950+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:46:18.749+0000] {processor.py:157} INFO - Started process (PID=6697) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:46:18.750+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:46:18.751+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:46:18.751+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:46:18.755+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:46:18.754+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:46:18.755+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:46:18.755+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:46:18.755+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:46:18.766+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:47:06.360+0000] {processor.py:157} INFO - Started process (PID=6753) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:47:06.362+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:47:06.363+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:47:06.362+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:47:06.367+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:47:06.366+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:47:06.367+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:47:06.367+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:47:06.367+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:47:06.379+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:47:56.218+0000] {processor.py:157} INFO - Started process (PID=6839) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:47:56.220+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:47:56.221+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:47:56.220+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:47:56.225+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:47:56.224+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:47:56.225+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:47:56.225+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:47:56.225+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:47:56.237+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:48:44.496+0000] {processor.py:157} INFO - Started process (PID=6893) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:48:44.497+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:48:44.498+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:48:44.498+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:48:44.502+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:48:44.501+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:48:44.503+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:48:44.502+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:48:44.503+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:48:44.514+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:49:35.370+0000] {processor.py:157} INFO - Started process (PID=6950) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:49:35.372+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:49:35.373+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:49:35.372+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:49:35.377+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:49:35.376+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:49:35.377+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:49:35.377+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:49:35.377+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:49:35.389+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:50:21.716+0000] {processor.py:157} INFO - Started process (PID=7001) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:50:21.717+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:50:21.718+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:50:21.718+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:50:21.723+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:50:21.722+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:50:21.723+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:50:21.723+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:50:21.723+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:50:21.735+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03T23:51:10.366+0000] {processor.py:157} INFO - Started process (PID=7056) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:51:10.368+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:51:10.369+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:51:10.368+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:51:10.373+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:51:10.372+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:51:10.373+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:51:10.373+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:51:10.373+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:51:10.385+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:51:57.482+0000] {processor.py:157} INFO - Started process (PID=7107) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:51:57.484+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:51:57.485+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:51:57.484+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:51:57.489+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:51:57.488+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:51:57.489+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:51:57.489+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:51:57.489+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:51:57.501+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:52:48.183+0000] {processor.py:157} INFO - Started process (PID=7163) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:52:48.185+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:52:48.186+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:52:48.185+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:52:48.190+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:52:48.189+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:52:48.190+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:52:48.190+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:52:48.190+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:52:48.202+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:53:38.494+0000] {processor.py:157} INFO - Started process (PID=7217) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:53:38.496+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:53:38.497+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:53:38.497+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:53:38.501+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:53:38.500+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:53:38.501+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:53:38.501+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:53:38.501+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:53:38.513+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:54:25.375+0000] {processor.py:157} INFO - Started process (PID=7270) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:54:25.377+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:54:25.378+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:54:25.377+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:54:25.382+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:54:25.381+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:54:25.382+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:54:25.382+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:54:25.382+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:54:25.394+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03T23:55:13.337+0000] {processor.py:157} INFO - Started process (PID=7322) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:55:13.339+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:55:13.340+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:55:13.339+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:55:13.344+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:55:13.343+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:55:13.344+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:55:13.344+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:55:13.344+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:55:13.356+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:56:00.947+0000] {processor.py:157} INFO - Started process (PID=7375) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:56:00.949+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:56:00.950+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:56:00.949+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:56:00.954+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:56:00.953+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:56:00.954+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:56:00.954+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:56:00.954+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:56:00.967+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03T23:56:51.348+0000] {processor.py:157} INFO - Started process (PID=7429) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:56:51.349+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:56:51.350+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:56:51.350+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:56:51.354+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:56:51.353+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:56:51.354+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:56:51.354+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:56:51.354+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:56:51.366+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:57:37.105+0000] {processor.py:157} INFO - Started process (PID=7482) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:57:37.107+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:57:37.108+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:57:37.107+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:57:37.111+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:57:37.111+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:57:37.112+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:57:37.112+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:57:37.112+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:57:37.123+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03T23:58:20.017+0000] {processor.py:157} INFO - Started process (PID=7530) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:58:20.019+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:58:20.020+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:58:20.019+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:58:20.024+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:58:20.023+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:58:20.024+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:58:20.024+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:58:20.024+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:58:20.036+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:59:06.781+0000] {processor.py:157} INFO - Started process (PID=7583) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:59:06.782+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:59:06.783+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:59:06.782+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:59:06.787+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:59:06.786+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:59:06.787+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:59:06.787+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:59:06.787+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:59:06.799+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03T23:59:53.919+0000] {processor.py:157} INFO - Started process (PID=7635) to work on /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:59:53.920+0000] {processor.py:829} INFO - Processing file /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03T23:59:53.921+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:59:53.920+0000] {dagbag.py:536} INFO - Filling up the DagBag from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:59:53.925+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:59:53.924+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03T23:59:53.925+0000] {logging_mixin.py:154} INFO - [2023-12-03T23:59:53.925+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03T23:59:53.925+0000] {processor.py:841} WARNING - No viable dags retrieved from /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03T23:59:53.937+0000] {processor.py:179} INFO - Processing /home/ubuntu/redfin/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
